documentation/commands/git-prune-branches.md

```diff
@@ -12,4 +12,6 @@ git prune-branches
 
 #### DESCRIPTION
 
-Deletes merged branches in the local and remote repositories.
+Deletes branches whose remote branch no longer exists,
```

Comment:
This appears to be the last place where "tracking" might be "better" then remote for consistency.